src/modules/NtpClient.rb

```diff
@@ -1118,14 +1112,9 @@ def write_chroot_config
       SCR.Write(path(".sysconfig.ntp"), nil)
     end
 
-    def update_netconfig
-      error = SCR.Execute(path(".target.bash"), "/sbin/netconfig update -m ntp") != 0
-
-      Report.Error(_("Cannot update the dynamic configuration policy.")) if error
-
-      !error
-    end
-
+    # Enable or disable ntp service depending on @run_service value. In case of
```

Comment:
I would prefer:

```
# Enable or disable ntp service depending on @run_service value
#
# * When disabling, it also stops the service.
# * When enabling, it tries to restart the service unless it's in write only mode.
```

Please, feel free to ignore this suggestion as it's a matter of personal preference.